tests/test_landings.py

```diff
@@ -47,7 +45,7 @@ def test_landing_revision_saves_data_in_db(db, client, phabfactory):
 
     # test saved data
     landing = Landing.query.get(1)
-    assert landing.serialize() == CANNED_LANDING_1
+    assert landing.serialize() == CANNED_LANDING_FACTORY_1
```

Comment:
you're right - it was to check if request id increments in stubbed transplant service